src/sentry/codecov/endpoints/repository_tokens/serializers.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+import sentry_sdk
+from rest_framework import serializers
+
+from sentry.codecov.endpoints.common.serializers import PageInfoSerializer
+
+logger = logging.getLogger(__name__)
+
+
+class RepositoryTokenNodeSerializer(serializers.Serializer):
+    """
+    Serializer for individual repository nodes from GraphQL response
+    """
+
+    name = serializers.CharField()
+    token = serializers.CharField()
+
+
+class RepositoryTokensSerializer(serializers.Serializer):
```

Comment:
we generally try to standardize on using a sentry response serializer, which can then work with our paginator classes. https://develop.sentry.dev/backend/api/serializers/#model-serializer.